src/entry-client.js

```diff
@@ -119,7 +121,6 @@ export default function initializeClient(createApp, clientOpts) {
         initialState: null,
         initialStateMetaTag: 'initial-state',
         vuexModules: true,
-        maxVuexModules: 2,
```

Comment:
Removing this option since the logic gets a good bit more complicated now that we not longer have a 1:1 route:module mapping.  Wasn't actively being used, so we can revisit later on if desired.